clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/RecoveryIntegrationSpec.scala

```diff
@@ -160,7 +160,7 @@ class RecoveryIntegrationSpec extends DockerIntegrationSpec with OptionValues {
                 )
                 .fold(throw _, identity)
 
-              s"$dateDir/${upsertId.persistenceFilename}" -> defaultPrinter.pretty(json)
+              s"$dateDir/${upsertId.persistenceFilename}" -> defaultPrinter.print(json)
```

Comment:
`print` or `printWith`?

Reply:
printWith is used with json objects. This is a string.